services/appmaker/helper.go

```diff
@@ -32,25 +34,25 @@ func containerCleanup(appName string) error {
 
 // diskCleanup cleans the specified application's container and local storage
 func diskCleanup(appName string) {
-	path, _ := os.Getwd()
 	appDir := filepath.Join(path, fmt.Sprintf("storage/%s", appName))
 	storeCleanupChan := make(chan error)
-	containerCleanupChan := make(chan error)
 	go func() {
 		storeCleanupChan <- storageCleanup(appDir)
 	}()
-	go func() {
-		containerCleanupChan <- containerCleanup(appName)
-	}()
+	containerCleanup(appName)
 	<-storeCleanupChan
```

Comment:
Why was the `containerCleanup` in a goroutine in the first place and why was it removed?

Reply:
Suppose you have a function `kekw` which calls 2 more functions `meow` and `bhow` in goroutines and blocks them
```
func kekw() {
go meow()
go bhow()
// block both meow and bhow via channels
}
```

But if kekw itself is called via a goroutine i.e `go kekw()` then the above function can be made equivalent to
```
func kekw() {
go meow()
bhow()
// block only meow via channel
}
```

In this case we are essentially saving a goroutine and a channel, hence better usage of cpu and memory